spark/src/main/scala/org/apache/spark/delta/sharing/PreSignedUrlCache.scala

```diff
@@ -63,6 +63,14 @@ class CachedTableManager(
     thread
   }
 
+  def isValidUrlExpirationTime(expiration: Long): Boolean = {
+    expiration > (System.currentTimeMillis() + refreshThresholdMs)
```

Comment:
Can you document why we need the extra refreshThresholdMs ?

Reply:
added.